src/scripts/settings/webSettings.js

```diff
@@ -72,7 +72,9 @@ export function getIncludeCorsCredentials() {
 
 export function getMultiServer() {
     return getConfig().then(config => {
-        return !!config.multiserver;
+        // Enable multi-server support in development environments
+        return process.env.NODE_ENV === 'development' // eslint-disable-line no-undef
```

Comment:
Don't we need to check that `process` is defined?
like this _(not tested)_
```suggestion
        return process?.env?.NODE_ENV === 'development' // eslint-disable-line no-undef
```

Reply:
Webpack replaces it with true/false at build time. I'm actually not sure if it would work with optional chaining lol. We use it in the serviceworker file already. https://github.com/jellyfin/jellyfin-web/blob/master/src/serviceworker.js#L49

Reply:
Wow, so it's like a preprocessor. But it also replaces `!!config.multiserver` to `0` :confused: 

I use dev's dist in place of installed to test webOS, but [this](https://github.com/jellyfin/jellyfin-web/blob/a70c83de3cffe65f99824e32b5d85b417ecade48/webpack.dev.js#L6) breaks targets.
So I am going to split dev config into `dev` (unminified, targeted on legacy) and `srv` (as it is now).

Is there a way to enable multi-server only for server mode? Somehow detect `serve`

Reply:
I can probably separate the conditions out so it's not replacing the config property also.

Reply:
I have updated this to use the define plugin to add a custom `WEBPACK_SERVE` variable replacement that we can use instead of the more generic `NODE_ENV`.

Reply:
I don't get how to change its value for #3027.
If var is not defined, webpack leaves it as is in the code (this can make a runtime error).
I tried to copy `DefinePlugin` to the common config with `false` value, but that from the dev config doesn't work anymore.

Reply:
:man_facepalming: yeah I should not have did it this way... it will also break in production builds since I did not add it there... I think using a single global variable would be better.